### PR TITLE
Update GL description on CHAOSS website

### PR DESCRIPTION
**Pull request by anajsana**
This commit adds new sections to grimoirelab.md, including:

* Main Components
* GrimoireLab tutorial
* Contributing to GrimoireLab
* Projects built using GrimoireLab

Signed-off-by: anajs <anitajs95@gmail.com>

### CODE REVIEW

**Comment by klumb** [comment]
This looks great! 

Thank you @anajsana

**Comment by klumb** [comment]
Issue #388

**Comment by anajsana** [comment]
happy to help @klumb :smile: !

File: Software/grimoirelab.md [hunk 1]
```diff
@@ -1,7 +1,56 @@
...
 
+GrimoireLab is focused on analyzing activity, community, and processes. Nonetheless, it can be easily tailored for other aims, and integrated with other tools.
+
+## Built with GrimoireLab
+
+Projects and services built using Grimoirelab technology:
+
+1. [Cauldron.io](https://cauldron.io/): A SaaS solution that allows project managers, analysts, and developers to understand more about the community and processes involved in software development. 
+2. [TLF Community Bridge Insights](https://lfanalytics.io/projects): A centralized platform that collects and visualizes data to let you monitor and analyze your open source projects.
+3. [Mautic Community Dashboard](https://dashboard.mautic.org/): The Community Dashboard uses the open-source GrimoireLab tool to import, analyze and visualize data from multiple sources in one central location.
```
Comment (GeorgLink) [inline Software/grimoirelab.md:13]: I we list Mautic, we should also list LibreOffice: https://dashboard.documentfoundation.org/

File: Software/grimoirelab.md [hunk 2]
```diff
@@ -1,7 +1,56 @@
...
+4. [Bitergia Analytics Platform](https://bitergia.com/bitergia-analytics/): A centralized source for metrics and data about software development projects.
+
+## Main components
+
+GrimoireLab tookit is organized in twelve different components. Each of them can be used as independent tools:
+
+**Data retrieval related components:**
```
Comment (GeorgLink) [inline Software/grimoirelab.md:20]:
```suggestion
**Data retrieval:**
```

File: Software/grimoirelab.md [hunk 3]
```diff
@@ -1,7 +1,56 @@
...
+
+* [Perceval](https://github.com/chaoss/grimoirelab-perceval): Tool used to retrieve and gather data from software repositories.
+* [Graal](https://github.com/chaoss/grimoirelab-graal): Source data analysis with external tools
+* [KingArthur](https://github.com/chaoss/grimoirelab-kingarthur): batch processing for massive retrieval
+
+**Data enrichment related components:**
```
Comment (GeorgLink) [inline Software/grimoirelab.md:26]:
```suggestion
**Data enrichment:**
```

File: Software/grimoirelab.md [hunk 4]
```diff
@@ -1,7 +1,56 @@
...
+
+* [GrimoireElk](https://github.com/chaoss/grimoirelab-elk): storage and enrichment of data
+* [SortingHat](https://github.com/chaoss/grimoirelab-sortinghat): identity management
+
+**Data visualization related components:**
```
Comment (GeorgLink) [inline Software/grimoirelab.md:31]:
```suggestion
**Data visualization:** 
```

File: Software/grimoirelab.md [hunk 5]
```diff
@@ -1,7 +1,56 @@
 # GrimoireLab
 
-GrimoireLab is a set of free, open source software tools for software development analytics. They gather data from many systems supporting development (git, GitHub, Jira, Bugzilla, Gerrit, mailing lists, Jenkins, Slack, Discourse, Confluence, StackOverflow, etc), merge and organize it in a database, and produce visualizations, actionable dashboards, and analytics of all of it.  GrimoireLab is focused on analyzing activity, community and processes, but can be easy tailored for other aims. They can also be easily integrated with other tools.
+GrimoireLab is a set of free, open source software tools for software development analytics. They gather data from several platforms involved in software development (Git, GitHub, Jira, Bugzilla, Gerrit, Mailing lists, Jenkins, Slack, Discourse, Confluence, StackOverflow, [and more](https://gitlab.com/Bitergia/c/FAQ/tree/master/sources#supported-data-sources)), merge and organize it in a database, and produce visualizations, actionable dashboards, and analytics of all of it.
```
Comment (GeorgLink) [inline Software/grimoirelab.md:3]: I suggest changing the link to GrimoireLab website. 

This PR would help: https://github.com/chaoss/grimoirelab/pull/393

**Review by GeorgLink — CHANGES_REQUESTED** [review]
Thanks @anajsana 

I have a few suggestions

File: Software/grimoirelab.md (hunk 5 above)
Comment (anajsana) [reply Software/grimoirelab.md]: ok! thanks for the feedback. Changes have been made @GeorgLink :+1:

**Comment by GeorgLink** [comment]
Thanks @anajsana

**Comment by klumb** [comment]
Thanks @anajsana

**Pull request merged**